Sources/Basics/FileSystem/NativePathExtensions.swift

```diff
@@ -23,6 +26,20 @@ extension AbsolutePath {
             return repr
         }
     }
+
+    public var _native: String {
```

Comment:
What's the difference between `_nativePathString` and `_native`? When would we use one and not the other? If we have to maintain both versions, would be better to rename them to make the distinction more obvious?

Reply:
also i would strongly prefer such extensions be on path and not on string

Reply:
@tomerd they are on `AbsolutePath`, not on `String`, or did I screw up the rebase?

@MaxDesiatov The `_native` might be better named `_normalized`, it is the path normalized as per the Windows path normalization rules, rather than the FSR of the path.  I'll rename it.

Reply:
What's FSR of the path? How's it defined? Can these details be added as doc comments to this file so that we can reference it later when working on it?

Reply:
Its the file system representation, it doesn't guarantee the normalization per se, only that it will convert the URL representation to what the file system expects.  I don't think we should be defining that, but mentioning that the function just returns the FSR vs the normalized path seems reasonable.